src/Core/Startup/ParameterConfirmationToken.php

```diff
@@ -157,6 +199,18 @@ public function reloadRequired()
     }
 
     /**
+     * Check if this token is provided either in the backurl, or directly,
+     * but without a token
+     *
+     * @return bool
+     */
+    public function reloadOnError()
```

Comment:
This method should match `reloadRequired`, so I've renamed to `reloadRequiredIfError` to be more explicit.